packages/commonjs/src/transform.js

```diff
@@ -299,6 +318,10 @@ export function transformCommonjs(
         return;
       }
 
+      // Is this a call to Object.defineProperty(exports, ...)?
+      const def = defineProperty(node, 'exports');
```

Comment:
I would rename the function to `isDefinePropertyCall`. Then the comment is also not needed. The return value could be `propertyName`, then I do not have to check the function definition what it means.

Reply:
done @lukastaegert :+1:

Reply:
actually on reflection I propose it be called `getDefinePropertyCallName`, the java programmer in me is complaining that an `isX` function should only return a boolean. See latest commit. That  said - happy to rollback if you prefer `isDefinePropertyCall`.

Reply:
Why not return a boolean?

Reply:
We could, but we’d also need to get the property name to add to namedExports?

Reply:
We could, but we’d also need to get the property name to add to namedExports?

Reply:
You are right, and you definitely got what I meant 😉

Reply:
Looks good. :shipit: